src/storage/src/hummock/sstable/block.rs

```diff
@@ -310,13 +416,22 @@ impl Block {
         self.restart_points.partition_point(pred)
     }
 
-    pub fn data(&self) -> &[u8] {
-        &self.data[..self.data_len]
+    pub fn slice(&self, range: impl RangeBounds<usize>) -> &[u8] {
```

Comment:
It looks strange to report the usage when calling slice, because slice is expected to be a very light weight method that only takes a slice. I think it's better to explicitly record the block hit when we advance the block iterator.

Reply:
+1 But i'm concerned if the caller forgets to record it.

Reply:
If we need to record each `slice()` call, let `slice()` records the statistic info sounds reasonable?

Reply:
I think it's too fined-grained to record each slice call. Shall we record the hit on maybe like restart point granularity?

Reply:
Also a good idea. Let me check.